### PR TITLE
Voice Over added into MainView and ListCard

### PR DESCRIPTION
**Pull request by luizmellodev**
Voice Over created into MainView and ListCard

### CODE REVIEW

File: Superlista/Service/DataService.swift [hunk 1]
```diff
@@ -40,6 +41,7 @@ class DataService: ObservableObject {
             var localLists = UDService().getUDLists()
             
             ckUserModel.sharedWithMe?.forEach { list in
+                print(list.name)
```
Comment (thaisrfernandes) [inline Superlista/Service/DataService.swift:44]: rm comment

File: Superlista/Views/MainView.swift [hunk 2]
```diff
@@ -87,6 +91,8 @@ struct MainView: View {
                             if !dataService.lists.isEmpty {
                                 Button(action: { isEditing.toggle() }, label: {
                                     Text(isEditing ? "MainViewTrailingNavigationLabelA": "MainViewTrailingNavigationLabelB")})
+                                    .accessibility(addTraits: .isButton)
```
Comment (thaisrfernandes) [inline Superlista/Views/MainView.swift:94]: rm trait

File: Superlista/Components/ListCard.swift [hunk 3]
```diff
@@ -149,6 +150,9 @@ struct ListCard: View {
                             dataService.currentList = list
                             showAlertDelete = true
                         }
+                        .accessibility(label: Text("Remover lista selecionada"))
```
Comment (thaisrfernandes) [inline Superlista/Components/ListCard.swift:153]: Criar localizables

File: Superlista/Components/ListCard.swift [hunk 4]
```diff
@@ -149,6 +150,9 @@ struct ListCard: View {
...
+                        .accessibility(hint: Text("Removerá a lista selecionada"))
```
Comment (thaisrfernandes) [inline Superlista/Components/ListCard.swift:154]: Criar localizables

File: pt-BR.lproj/Localizable.strings [hunk 5]
```diff
@@ -112,6 +119,7 @@
 "Option4Hint" = "Toque para compartilhar lista";
 "Option5" = "Deletar lista";
 "Option5Hint" = "Toque para deletar lista";
+"HintListCard" = "Abrir lista";
```
Comment (thaisrfernandes) [inline pt-BR.lproj/Localizable.strings:123]: Adicionar localizables em todos os arquivos.

**Review by thaisrfernandes — CHANGES_REQUESTED** [review]

File: Superlista/Components/ListCard.swift (hunk 3 above)
Comment (luizmellodev) [reply Superlista/Components/ListCard.swift:153]: feito 🆗

**Review by thaisrfernandes — APPROVED** [review]

**Pull request merged**